pkgs/development/python-modules/python-youtube/default.nix

```diff
@@ -47,24 +45,6 @@ buildPythonPackage rec {
     responses
   ];
 
-  disabledTests = [
-    # On both tests, upstream compares a string to an integer
-
-    /*
-      python3.10-python-youtube> >       self.assertEqual(m.viewCount, "160361638")
-      python3.10-python-youtube> E       AssertionError: 160361638 != '160361638'
-      python3.10-python-youtube> tests/models/test_channel.py:62: AssertionError
-    */
-    "testChannelStatistics"
-
-    /*
-      python3.10-python-youtube> >       self.assertEqual(m.viewCount, "8087")
-      python3.10-python-youtube> E       AssertionError: 8087 != '8087'
-      python3.10-python-youtube> tests/models/test_videos.py:76: AssertionError
-    */
-    "testVideoStatistics"
-  ];
-
   meta = with lib; {
     description = "A simple Python wrapper around for YouTube Data API";
```

Comment:
Can we tell upstream to fix the typo?

Reply:
https://github.com/sns-sdks/python-youtube/issues/161